app/assets/javascripts/discourse/app/mixins/uppy-upload.js

```diff
@@ -221,6 +226,16 @@ export default Mixin.create(UppyS3Multipart, {
     }
   },
 
+  _startUpload() {
+    if (!this.filesAwaitingUpload) {
+      return;
+    }
+    if (!this._uppyInstance.getFiles().length) {
+      return;
+    }
+    return this._uppyInstance?.upload();
```

Comment:
Minor nit: since you're using `_uppyInstance` without `?.` above, it means it either will fail earlier or this `?.` isn't needed